lib/prisma-client-exception.filter.ts

```diff
@@ -87,20 +101,51 @@ export class PrismaClientExceptionFilter extends BaseExceptionFilter {
       );
     } else if (host.getType<GqlContextType>() === 'graphql') {
       // for graphql requests
-      if (!Object.keys(this.errorCodesStatusMapping).includes(exception.code)) {
+      if (statusCode === undefined) {
         return exception;
       }
 
       return new HttpException({ statusCode, message }, statusCode);
     }
   }
 
-  private exceptionShortMessage(message: string): string {
-    const shortMessage = message.substring(message.indexOf('→'));
-    return shortMessage
-      .substring(shortMessage.indexOf('\n'))
-      .replace(/\n/g, '')
-      .trim();
+  private userDefinedStatusCode(
+    exception: Prisma.PrismaClientKnownRequestError,
+  ): number | undefined {
+    const userDefinedValue = this.userDefinedMapping?.[exception.code];
+    return typeof userDefinedValue === 'number'
+      ? userDefinedValue
+      : userDefinedValue?.statusCode;
+  }
+
+  private defaultStatusCode(
+    exception: Prisma.PrismaClientKnownRequestError,
+  ): number | undefined {
+    return this.defaultMapping?.[exception.code];
```

Comment:
`defaultMapping` is always available

```suggestion
    return this.defaultMapping[exception.code];
```